web/src/lib/components/shared-components/leaflet/asset-marker-cluster.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts" context="module">
+	import { createContext } from '$lib/utils/context';
+
+	const { get: getContext, set: setClusterContext } = createContext<() => L.MarkerClusterGroup>();
+
+	export const getClusterContext = () => {
+		return getContext()();
+	};
+</script>
+
+<script lang="ts">
+	import { onDestroy, onMount } from 'svelte';
+	import 'leaflet.markercluster';
+	import { getMapContext } from './map.svelte';
+	import { AssetResponseDto } from '@api';
+	import { Marker, Icon } from 'leaflet';
+
+	export let assets: AssetResponseDto[];
+
+	const map = getMapContext();
+	let cluster: L.MarkerClusterGroup;
+
+	setClusterContext(() => cluster);
+
+	onMount(() => {
+		cluster = new L.MarkerClusterGroup({
+			showCoverageOnHover: false,
+			zoomToBoundsOnClick: true,
+			spiderfyOnMaxZoom: true,
+			maxClusterRadius: 30,
+			spiderLegPolylineOptions: { opacity: 0 }
+		});
+
+		for (let asset of assets) {
+			if (asset.exifInfo) {
```

Comment:
In the negative cases can we just continue and unindent everything below the if?